set.mm

```diff
@@ -788149,6 +789198,62 @@ Nonzero rings (extension)
       WHWLRWKXTXMYFWCWSXLYEJBXSXKKBXSWMWNUTWKVOWDJKBDCXEEXIGUUDUUAWOWP $.
   $}
 
+  ${
+    $d F x $.  $d R x $.  $d S x $.
+    rngisom1.1 $e |- .1. = ( 1r ` R ) $.
+    rngisom1.b $e |- B = ( Base ` S ) $.
+    $( If there is a non-unital ring isomorphism between a unital ring and a
+       non-unital ring, then the function value of the ring unity of the unital
+       ring is an element of the base set of the non-unital ring.  (Contributed
+       by AV, 27-Feb-2025.) $)
+    rngisomfv1 $p |- ( ( R e. Ring /\ F e. ( R RngIsom S ) )
+                       -> ( F ` .1. ) e. B ) $=
+      ( crg wcel crngs co wa cbs cfv wf wf1o eqid rngimf1o f1of syl ffvelcdmd
+      adantl ringidcl adantr ) BHIZEBCJKIZLBMNZADEUFUGAEOZUEUFUGAEPUHUGABCEUGQZ
+      GRUGAESTUBUEDUGIUFUGBDUIFUCUDUA $.
+
+    rngisom1.t $e |- .x. = ( .r ` S ) $.
+    $( If there is a non-unital ring isomorphism between a unital ring and a
+       non-unital ring, then the function value of the ring unity of the unital
+       ring is a ring unity of the non-unital ring.  (Contributed by AV,
+       27-Feb-2025.) $)
+    rngisom1 $p |- ( ( R e. Ring /\ S e. Rng /\ F e. ( R RngIsom S ) )
+                     -> A. x e. B ( ( ( F ` .1. ) .x. x ) = x
+                                    /\ ( x .x. ( F ` .1. ) ) = x ) ) $=
+      ( wcel co cfv wceq wa syl 3ad2ant3 adantr syl3anc fveq2d crg crngs w3a cv
+      crng ccnv cmulr crngh rngimcnv cbs rngimrnghm rngisomfv1 3adant2 rnghmmul
+      eqid simpr rngimf1o simpl2 rngcl f1ocnvfv2 syl2an2r ringidcl 3ad2ant1 jca
+      wf1o f1ocnvfv1 oveq1d simpl1 f1of ffvelcdmda ringlidmd eqtrd sylan oveq2d
+      wf 3eqtr3d ringridmd 3eqtrd ralrimiva ) CUAKZDUEKZGCDUBLKZUCZFGMZAUDZELZW
+      ENZWEWDELZWENZOABWCWEBKZOZWGWIWKWFGUFZMZGMZWDWLMZWEWLMZCUGMZLZGMZWFWEWKWM
+      WRGWKWLDCUHLKZWDBKZWJWMWRNWCWTWJWBVTWTWAWBWLDCUBLKZWTCDGUIZBCUJMZDCWLIXDU
+      OZUKZPQRWCXAWJVTWBXAWABCDFGHIULUMRZWCWJUPZWDWEDCEWQWLBIJWQUOZUNSTWCXDBGVE
+      ZWJWFBKZWNWFNWBVTXJWAXDBCDGXEIUQQZWKWAXAWJXKVTWAWBWJURZXGXHBDEWDWEIJUSSXD
+      BWFGUTVAWKWSWPGMZWEWKWRWPGWKWRFWPWQLWPWKWOFWPWQWKXJFXDKZOZWOFNWCXPWJWCXJX
+      OXLVTWAXOWBXDCFXEHVBVCVDRXDBFGVFPZVGWKXDCWQFWPXEXIHVTWAWBWJVHZWCBXDWEWLWB
+      VTBXDWLVOZWAWBXBXSXCXBBXDWLVEXSBXDDCWLIXEUQBXDWLVIPPQVJZVKVLTWCXJWJXNWENX
+      LXDBWEGUTVMZVLVPWKWHWLMZGMZXNWHWEWKYBWPGWKYBWPWOWQLZWPFWQLWPWKWTWJXAYBYDN
+      WCWTWJWCXBWTWBVTXBWAXCQXFPRXHXGWEWDDCEWQWLBIJXIUNSWKWOFWPWQXQVNWKXDCWQFWP
+      XEXIHXRXTVQVRTWCXJWJWHBKZYCWHNXLWKWAWJXAYEXMXHXGBDEWEWDIJUSSXDBWHGUTVAYAV
+      PVDVS $.
+  $}
```

Comment:
These theorems look more like lemmas for `~ rngisomring`.

Reply:
They curently are, indeed. But I think they are also interesting in their own right, and may be used in other contexts later.

Reply:
Yes, who knows?